_episodes/02-filedir.md

```diff
@@ -348,21 +356,18 @@ shell-lesson-data/
 
 Note that if a directory named `Desktop` does not exist in your current working directory,
 this command will return an error. Typically, a `Desktop` directory exists in your
-home directory, which we assume is the current working directory of your bash shell.
+home directory, which we assume is the current working directory of your shell.
 
 Your output should be a list of all the files and sub-directories in your
 Desktop directory, including the `shell-lesson-data` directory you downloaded at
-the [setup for this lesson]({{ page.root }}{% link setup.md %}).
-On many systems,
-the command line Desktop directory is the same as your GUI Desktop.
-Take a look at your Desktop to confirm that your output is accurate.
-
-As you may now see, using a bash shell is strongly dependent on the idea that
-your files are organized in a hierarchical file system.
-Organizing things hierarchically in this way helps us keep track of our work:
-it's possible to put hundreds of files in our home directory,
-just as it's possible to pile hundreds of printed papers on our desk,
-but it's a self-defeating strategy.
+the [setup for this lesson]({{ page.root }}{% link setup.md %}). (On most systems, the
+contents of the `Desktop` directory in the shell will show up as icons on the
+workspace behind all the open windows. See if this is the case for you.)
+
+Organizing things hierarchically in this way helps us keep track of our work. While it's
+it's possible to put hundreds of files in our home directory just as it's possible to
+pile hundreds of printed papers on our desk, it's much easier to find things when
+they've been organized into sensibly-named subdirectories.
```

Comment:
This is much clearer. Thanks. Minor suggestion to emphasize GUI since it is possible to run without one
```suggestion
the [setup for this lesson]({{ page.root }}{% link setup.md %}). (On most systems, the
contents of the `Desktop` directory in the shell will show up as icons in a graphical
user interface behind all the open windows. See if this is the case for you.)

Organizing things hierarchically in this way helps us keep track of our work. While it's
it's possible to put hundreds of files in our home directory just as it's possible to
pile hundreds of printed papers on our desk, it's much easier to find things when
they've been organized into sensibly-named subdirectories.
```